examples/tests/18-e2sm.rs

```diff
@@ -5,5 +5,18 @@ mod e2sm {
 }
 
 fn main() {
+    use asn1_codecs::{aper::AperCodec, PerCodecData};
     eprintln!("E2SM");
+
+    // This will break: Fixed
+    let decode_str = "080000000100000000002043514901200000";
+
+    // This one works if it is uncommented
```

Comment:
I'd remove this comment line and the one below it... they are confusing to those without context as long as your changes work.